robolectric/src/main/java/org/robolectric/RobolectricTestRunner.java

```diff
@@ -511,7 +512,10 @@ private Configuration getConfiguration(Method method) {
    *     details.
    */
   @Deprecated
-  protected Config buildGlobalConfig() {
+  @InlineMe(
+      replacement = "new Config.Builder().build()",
+      imports = "org.robolectric.annotation.Config")
+  protected final Config buildGlobalConfig() {
```

Comment:
@hellosagar could you remove those added final modifier? I am afraid that there are some users extend those classes and reimplement those methods, and the added  final maybe break their code.

Reply:
ah Okay, Sure 👍🏻

Reply:
but in that case we cant inline it because overridable methods are not inlined.

```
/Users/sagarkhurana/AndroidStudioProjects/robolectric/robolectric/src/main/java/org/robolectric/RobolectricTestRunner.java:518: error: [InlineMeValidator] Methods that are inlined should not be overridable, as the implementation of an overriding method may be different than the inlining
  protected Config buildGlobalConfig() {
                   ^
    (see https://errorprone.info/bugpattern/InlineMeValidator)
  Did you mean to remove this line?
  ```
  
  so should I remove `@InlineMe` annotation and suppress it?

Reply:
In this case, I recommend to supress it.

Reply:
Okay